plugins/contact-resources/src/components/UserBoxItems.svelte

```diff
@@ -64,6 +68,10 @@
     )
   }
 
+  function getPersons (employees: Ref<Employee>[]) {
```

Comment:
you need to pass employeeByIdStore as parameter, over wise svelte will not update view if new employee will arrive.